nativelib/src/main/resources/scala-native/gc/commix/datastructures/GreyPacket.c

```diff
@@ -34,15 +34,15 @@ void GreyList_Init(GreyList *list) {
     list->head.atom = GreyPacketRef_Empty();
 }
 
-uint32_t GreyList_Size(GreyList *list) {
+UInt24 GreyList_Size(GreyList *list) {
```

Comment:
It seems all the callers of `GreyList_Size` immediately convert it to `uint32_t`. I suggest keeping the result type of `uint32_t` here, and doing the conversion in this function.